content/src/helpers/FetchHelper.ts

```diff
@@ -40,4 +40,18 @@ export class FetchHelper {
         }
     }
 
+}
+
+export async function retry<T>(execution: () => Promise<T>, attempts: number, description: string, waitTime: string = '1s'): Promise<T> {
+    while (attempts > 0) {
+        try {
+            return await execution()
+        } catch (error) { }
```

Comment:
I think it's important to report the cause of the error, maybe at the last attempt.

Reply:
Done!